settings/ui/components/SettingsApp.vue

```diff
@@ -15,32 +15,33 @@
                 {{ selectedExtensionName }}
               </h1>
             </div>
-            <hr />
+            <hr/>
           </template>
           <template v-if="settingsValuesLoaded">
             <settings-bundle
-              v-for="bundle in selectedBundles"
-              :key="'bundle-' + bundle.id"
-              :bundle="bundle"
-              class="oc-mt"
+                v-for="bundle in selectedBundles"
```

Comment:
The indentation looks wrong here and in the next three lines 🤔

Reply:
dang, my formatter haven't picked this up... done in new commit.